docs/pages/upgrading/agent-managed-updates-v1.mdx

```diff
@@ -1,31 +1,31 @@
 ---
-title: Teleport Managed Agent Updates (v1)
-description: Describes how to set up Teleport Managed Agent Updates v1.
+title: Managed Updates for Teleport Agents (v1)
+description: Describes how to set up Managed Updates for Teleport Agents (v1)
 ---
 
 <Notice type="warning">
-This document describes the Managed Updates v1 Teleport Agent updater, which is
-currently supported but will be deprecated after the [Managed update v2 updater
-](./agent-managed-updates.mdx) is generally available.
+This document describes Managed Updates for Agents (v1), which is
+currently supported but will be deprecated after [Managed Updates for
+Agents (v2)](./agent-managed-updates.mdx) is generally available.
 </Notice>
 
-The Managed Updates v1 updater uses a script called `teleport-upgrade` that is provided by
+Managed Updates v1 uses a script called `teleport-upgrade` that is provided by
 the `teleport-ent-updater` package and configured by the `cluster_maintenance_config`
-Teleport resource. The new updater uses a binary called `teleport-update` that is
+Teleport resource. Managed Updates v2 uses a binary called `teleport-update` that is
 provided by the `teleport` package and configured by the `autoupdate_version` and
 `autoupdate_config`. The original updater and resource are described in this document.
 
 Only Enterprise versions of Teleport can use Managed Updates v1.
 
 Please consider using [Managed Updates for Agents (v2)](./agent-managed-updates.mdx),
-as its updater provides a safer, simpler, more flexible, compatible, and reliable
-update experience compared to Managed Updates v1.
+as it provides a safer, simpler, more flexible, compatible, and reliable update experience
+compared to Managed Updates v1.
 
 <Admonition type="note" title="Compatibility between Managed Updates v1 and v2">
-Managed Updates v2 are backwards compatible with the
+Managed Updates v2 is backwards-compatible with the
```

Comment:
As Managed Updates are a plural product name, I would expect the rest of the sentence to use plural as well.

@ptgott WDYT?

Reply:
I think it's a singular title name, like 'Cats is a musical'

Reply:
Since it's a name, I think the singular makes sense to me, especially with the version as part of the name.